lib/Qi.pm

```diff
@@ -20,9 +26,17 @@ sub startup {
 
   # Normal route to controller
   $r->get('/')->to('example#welcome');
+  
+  my $api = $r->under('/api');
+  $api->get('/assets')->to('assets#get')->name('get');       # get assets
+  $api->post('/assets')->to('assets#insert')->name('test');     # insert asset
+  $api->put('/assets')->to('assets#update')->name('update');     # update asset
+  $api->delete('/assets')->to('assets#remove')->name('remove'); # remove asset
+
```

Comment:
These four routes above a great!  But use more descriptive names (and of course "test" is not correct.  :)

Examples of better names: assets_get, assets_insert, etc...  Keep in mind, you'll end up having route for another CRUD table at some point.  They can't all be called "get"!